src/assets/css/_main.scss

```diff
@@ -135,7 +137,7 @@ a:focus {
 
 .mzp-c-footer-links-social a:focus,
 .mzp-c-footer-primary-logo a:focus,
-.topNav a:focus,
+.top-nav a:focus,
```

Comment:
I tried to avoid most of these (filed https://github.com/mozilla/extension-workshop/issues/1219) to come back and grab the rest. However, there were a few that had only a small number of views using them so figured I'd change the bits that didn't touch too many places.

Happy to revert for now though if folks would prefer we do this as a separate PR.